packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -884,6 +885,27 @@ export const useArbTokenBridge = (
     return rec
   }
 
+  function addL2NativeToken(erc20L2Address: string) {
+    const token = getL2NativeToken(erc20L2Address, l2.network.id)
```

Comment:
the `getL2NativeToken` might throw an error
use a try catch here instead of when calling `addL2NativeToken`?

Reply:
No, I want it to throw an error when it can't be added as an L2-native token, and have it be handled in the call site